### PR TITLE
New 2020 census page

### PR DESCRIPTION
**Pull request by gregbunce**
checking how everything looks. 

also: i'll be sharing the preview link with the folks at the state data center to see if they want any changes.

### CODE REVIEW

**Comment by gregbunce** [comment]
this pr is ready for review.

File: data/demographic/census/index.html [hunk 1]
```diff
@@ -59,7 +77,36 @@
...
+abstract="This page contains boundary and demographic data collected by the U.S. Census Bureau for 2020, 2010 and 2000 censuses. Demographic data includes population counts, household counts, ethnicity counts, etc. The data available on this page is filtered, geographically, to the state of Utah. This data follows the format released by the Census Bureau, with just a few modifications for usability." %}
+<p>Read about <a href="{% link data/demographic/census/index.html %}#Commonly-Used Census Bureau Geography">commonly-used Census Bureau geography</a> including blocks, block groups, tracts, and places.</p>
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+      <h3 id="2020Census">2020 U.S. Census Bureau Data</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+      <img class="inline-text-left outline" src="{% link images/censussmall.png %}" alt="2020 U.S. Census Bureau Data" loading="lazy" />
```
Comment (steveoh) [inline data/demographic/census/index.html:87]: **praise**: thanks for the lazy!

File: data/demographic/census/index.html [hunk 2]
```diff
@@ -59,7 +77,36 @@
...
+      <p>The data in this section contains census blocks, census block groups, census tracts, census places, census counties, and the state boundary from the Census Bureau.</p>
+      <p>Utah’s 2020 census geographic boundaries became available in February 2021. The Census Bureau will release the first set of corresponding demographic data in September 2021 (the 2020 Census Redistricting P.L. 94-171 Summary Files). Following that release, AGRC will append the demographic data to the existing 2020 geographies served on this page.</p>
```
Comment (steveoh) [inline data/demographic/census/index.html:89]: not sure if this reads better or not
```suggestion
      <p>Utah’s 2020 census geographic boundaries became available in February 2021. The Census Bureau will release the first set of corresponding demographic data in September 2021 (the 2020 Census Redistricting P.L. 94-171 Summary Files). Following that release, AGRC will merge the demographic data with the existing 2020 data available now.</p>
```

File: data/demographic/census/index.html [hunk 3]
```diff
@@ -59,7 +77,36 @@
...
+      <p>The 2020 data available on this page was downloaded from the <a href="https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2">2020 Census Redistricting Data (P.L. 94-171)</a> page. All 2020 census boundaries are current to January 1, 2020. Additional GIS files and support products are available on the <a href="https://www.census.gov/programs-surveys/decennial-census/about/rdo/summary-files.html#P2">census redistricting data</a> page.</p>
+      <p>Geography is a fundamental aspect of the census, providing the framework for the once-a-decade count of population and housing. Read about some of the key geography changes in the Kem C. Gardner Policy Institute's <a href="https://gardner.utah.edu/blog-2020-census-geographic-support-products-available/">2020 census geography blog</a>.</p>
```
Comment (steveoh) [inline data/demographic/census/index.html:91]:
```suggestion
      <p>Geography is a fundamental aspect of the census, providing the framework for the once-a-decade count of population and housing. There are key geography changes that can be read about in the Kem C. Gardner Policy Institute's <a href="https://gardner.utah.edu/blog-2020-census-geographic-support-products-available/">2020 census geography blog</a>.</p>
```

File: data/demographic/census/index.html [hunk 4]
```diff
@@ -59,7 +77,36 @@
...
+      <p class="pop">Note: AGRC is currently loading the 2020 census data into the SGID where it will be available to consume as web map services or direct database access. In the meantime, this data is available for download now as individual shapefiles or packaged in a file geodatabase. See the download links below.</p>
+      <p>There are no constraints or warranties with regard to the use of this dataset.</p>
```
Comment (steveoh) [inline data/demographic/census/index.html:93]: **question**: should this link to the disclaimer instead?

File: data/demographic/census/index.html [hunk 5]
```diff
@@ -149,3 +196,15 @@ <h5>Related Resources</h5>
   {% include packagedata.html name="UrbanAreasCensus2010"
   info=page.UrbanAreasCensus2010 %}
 </div>
+<h2 id="Commonly-Used Census Bureau Geography">Definitions for Commonly-Used Geography</h2>
```
Comment (steveoh) [inline data/demographic/census/index.html:199]: **thought**: not sure about spaces in id's
```suggestion
<h2 id="common-definitions">Definitions for Commonly-Used Geography</h2>
```

File: data/demographic/census/index.html [hunk 6]
```diff
@@ -59,7 +77,36 @@
...
 name="Demographic Data"
 stewards="AGRC and U.S. Census Bureau"
-abstract="This data package contains demographic information collected by the Census Bureau in 2010 and 2000. Demographic population includes population counts, household counts, ethnicity counts, etc.  Below, the U.S. Census Bureau Data is the data in the format released by the Census Bureau, with just a few modifications to make the data more user friendly. The Populated Block Areas 2010 Approximation and Populated Place Points 2010 Approximation are datasets derived by AGRC from 2010 Census data." %}
+abstract="This page contains boundary and demographic data collected by the U.S. Census Bureau for 2020, 2010 and 2000 censuses. Demographic data includes population counts, household counts, ethnicity counts, etc. The data available on this page is filtered, geographically, to the state of Utah. This data follows the format released by the Census Bureau, with just a few modifications for usability." %}
+<p>Read about <a href="{% link data/demographic/census/index.html %}#Commonly-Used Census Bureau Geography">commonly-used Census Bureau geography</a> including blocks, block groups, tracts, and places.</p>
```
Comment (steveoh) [inline data/demographic/census/index.html:81]: **thought**: i wasn't sure about spaces in anchor tags and i'll assume you tested this (and can dismiss these comments) but if not these two suggestions use a simpler anchor.
```suggestion
<p>Read about <a href="{% link data/demographic/census/index.html %}#common-definitions">commonly-used Census Bureau geography</a> including blocks, block groups, tracts, and places.</p>
```

**Review by steveoh — APPROVED** [review]

**Pull request merged**